crates/phala-types/src/lib.rs

```diff
@@ -512,7 +512,9 @@ pub mod messaging {
     pub enum GatekeeperEvent {
         NewRandomNumber(RandomNumberEvent),
         TokenomicParametersChanged(TokenomicParameters),
+        /// Deprecated after https://github.com/Phala-Network/phala-blockchain/pull/499
         RepairV,
```

Comment:
I suggest we add `_` prefix for depracated enum like `_RepairV` which more self-explanation

Reply:
It's still in used in the code, but just shouldn't be triggered in the future